tensorboard/compat/proto/update.sh

```diff
@@ -39,8 +39,10 @@ find tensorboard/compat/proto/ -type f  -name '*.proto' -exec perl -pi \
   -e 's|tensorflow\.DataType|tensorboard.DataType|g;' \
   -e 's|tensorflow\.TensorProto|tensorboard.TensorProto|g;' \
   -e 's|tensorflow\.TensorShapeProto|tensorboard.TensorShapeProto|g;' \
+  -e 's|LINT\.|DISABLED.|g;' \
```

Comment:
Could we narrow this matching a little bit, for example like this?

```
s|^// LINT\.|// DISABLED.|g;
```

That way it will only match `LINT.` if it's at the start of the line and preceded by `// `.